### PR TITLE
Fix waiting for Helper timeout

### PR DESCRIPTION
**Pull request by dainnilsson**
This changes how we wait for the Helper process, indicating that it is slow after a sufficient delay, but still recovering after the delay if it becomes responsive.
This also introduces delayed visibility of AppPage, which is used to delay the showing of the CircularProgressIndicators when the page is only displayed for a split second.

### CODE REVIEW

**Review by AdamVe — APPROVED** [review]
Improves user experience. Tested on Mac OS and Android and did not encountered any issues.

File: lib/app/views/device_error_screen.dart [hunk 1]
```diff
@@ -48,7 +48,7 @@ class DeviceErrorScreen extends ConsumerWidget {
                     context, 'Elevating permissions...',
                     duration: const Duration(seconds: 30));
                 try {
-                  if (await ref.read(rpcProvider).elevate()) {
+                  if (await ref.read(rpcProvider).value!.elevate()) {
```
Comment (fdennis) [inline lib/app/views/device_error_screen.dart:51]: Is there a reason for having `value!` here instead of `requireValue`?

Comment (dainnilsson) [reply lib/app/views/device_error_screen.dart:51]: There isn't, good catch!

File: lib/desktop/init.dart [hunk 2]
```diff
@@ -145,38 +141,42 @@ Future<Widget> initialize(List<String> argv) async {
...
       )
     ],
     child: YubicoAuthenticatorApp(
       page: Consumer(
-        builder: ((_, ref, child) {
+        builder: ((context, ref, child) {
           // keep RPC log level in sync with app
           ref.listen<Level>(logLevelProvider, (_, level) {
-            rpc.setLogLevel(level);
+            ref.read(rpcProvider).valueOrNull?.setLogLevel(level);
```
Comment (fdennis) [inline lib/desktop/init.dart:155]: I'm also wondering if this should be changed to `requireValue` as well?

Comment (dainnilsson) [reply lib/desktop/init.dart:155]: No, in this case it is correct. We don't want to fail if this happens before the rpcProvider is initialized, so null here is fine (we'll just not call setLogLevel). `requireValue` would thrown an exception in that case.

**Review by fdennis — APPROVED** [review]

File: lib/desktop/init.dart (hunk 2 above)
Comment (fdennis) [reply lib/desktop/init.dart:155]: Alright, cool!

**Pull request merged**